core/src/main/java/org/opensearch/sql/expression/DSL.java

```diff
@@ -735,6 +735,18 @@ public static Aggregator max(Expression... expressions) {
     return aggregate(BuiltinFunctionName.MAX, expressions);
   }
 
+  /**
+   * OpenSearch uses T-Digest to approximate percentile, so PERCENTILE and PERCENTILE_APPROX are the
+   * same function.
+   */
+  public static Aggregator percentile(Expression... expressions) {
```

Comment:
is it one-way door decision to make percentile same as percentile_approx? should percnetile means accurate accurate?

Reply:
There is no exact percentile calculation in OpenSearch Core. There could be two options for it:
- Option 1:
1. Adding functions `percentile(field, p)` and `percentile_approx(field, p)` as **approximate** calculation (in this PR)
2. Adding ANSI SQL functions `percentile_cont(p) WITHIN GROUP (ORDER BY field)` and `percentile_disc(p) WITHIN GROUP (ORDER BY field)` as **exact** calculation when OpenSearch Core has supported (in follow-ups)

- Option 2:
1. Adding function `percentile_approx(field, p)` as **approximate** calculation (in this PR)
2. Adding functions `percentile(field, p)` and ANSI SQL functions `percentile_cont(p) WITHIN GROUP (ORDER BY field)` and `percentile_disc(p) WITHIN GROUP (ORDER BY field)` as **exact** calculation when OpenSearch Core has supported, the  `percentile(field, p)`  is same with `percentile_cont` (in follow-ups)

Option 1 will be more friendly for the user who is used to using OpenSearch DSL or Elasticsearch.
Option 2 is what Spark did since `percentile(field, p)` is a Hive compatible function in Spark.
Any throughs?

Reply:
thanks for the explaination. I perfer option 1.